tests/core/test_subdir_data.py

```diff
@@ -274,8 +279,9 @@ def __enter__(self):
         def __exit__(self, *exc):
             Channel._cache_.clear()
 
+    # disable SSL_VERIFY to cover 'turn off warnings' line
     with ChannelCacheClear(), make_temp_env(), env_vars(
-        {"CONDA_PLATFORM": platform},
+        {"CONDA_PLATFORM": platform, "CONDA_SSL_VERIFY": "false"},
```

Comment:
I'm not sure if I understand, what's "'turn off warnings' line"?

Reply:
We want to run this branch at least once for code coverage https://github.com/conda/conda/blob/main/conda/gateways/repodata/__init__.py#L78

```
    def repodata(self, state: RepodataState) -> str | None:
        if not context.ssl_verify:
            warnings.simplefilter("ignore", InsecureRequestWarning)
```